src/components/Homepage/Homepage.js

```diff
@@ -0,0 +1,44 @@
+import createElement from '../../lib/createElement'
+import './Homepage.css'
+import Card from '../Card'
+
+export default function Homepage(characters) {
+  const el = createElement('main', { className: 'Homepage' })
+
+  function setCards(characters) {
+    const cardElements = characters.map(
+      ({
+        name,
+        house,
+        image,
+        gender,
+        yearOfBirth,
+        ancestry,
+        patronus,
+        wand,
+        hogwartsStudent,
+        hogwartsStaff,
+        alive,
+      }) =>
+        Card(
+          name,
+          house,
+          image,
+          gender,
+          yearOfBirth,
+          ancestry,
+          patronus,
+          wand,
+          hogwartsStudent,
+          hogwartsStaff,
+          alive
+        )
```

Comment:
```suggestion
    const cardElements = characters.map(character => Card(...character))
```

or if you change `Card` as suggested above (do destructuring assignment of a single parameter in `Card`):

```suggestion
 const cardElements = characters.map(character => Card(character))
```